securedrop/template_filters.py

```diff
@@ -10,7 +10,7 @@ def datetimeformat(dt, fmt=None, relative=False):
     if relative:
         time_difference = _relative_timestamp(dt)
         if time_difference:
-            return gettext('{time} ago').format(time=time_difference)
```

Comment:
I'm preparing a pull request to use http://babel.pocoo.org/en/latest/dates.html instead of these hand made functions so we don't have to translate. I should have thought about it earlier but ... :-)

Reply:
Since we control this repo, we should merge this in now to have the fix and then roll it back once upstream makes the fix.

Reply:
Sorry for the confusion: I mean I'm preparing a pull request against SecureDrop not against babel. The dates functions of babel work fine and we can delete this code. Less code less bugs !

Reply:
Ok, well I'm going to leave this in for now. When you make your PR, it will be a race to see who can get it merged first :)

Reply:
Here you go https://github.com/freedomofpress/securedrop/pull/2353 the race is on :-D